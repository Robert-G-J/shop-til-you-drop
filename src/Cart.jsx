import React from "react";
import FoodItemRow from "./FoodItemRow";

const Cart = () => (
  <body>
    <header className="header">
      <h1 className="header__title">Shopping Cart</h1>
    </header>
    <FoodItemRow />
  </body>
);

export default Cart;
