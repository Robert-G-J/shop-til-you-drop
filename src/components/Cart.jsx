import React from "react";
import ProductTableContainer from "./ProductTableContainer";

const Cart = () => (
  <div className="cart__body">
    <header className="header">
      <h1 className="header__title">Rick's Kit Cart</h1>
    </header>
    <ProductTableContainer />
  </div>
);

export default Cart;
