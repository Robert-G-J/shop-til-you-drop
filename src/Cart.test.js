/* eslint-disable */

import React from "react";
import Cart from "./Cart";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("The shopping cart app", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Cart />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("doesn't crash", () => {
    const component = shallow(<Cart />);
    expect(component.exists()).toEqual(true);
  });
});
