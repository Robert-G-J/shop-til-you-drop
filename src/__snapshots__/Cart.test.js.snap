// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The shopping cart app renders correctly 1`] = `
<body>
  <header
    className="header"
  >
    <h1
      className="header__title"
    >
      Shopping Cart
    </h1>
  </header>
  <div
    className="food-item-row"
  >
    <div
      className="food-item-row__item"
    />
    <div
      className="food-item-row__price"
    />
    <button
      className="button-increment"
    >
      +
    </button>
    <button
      className="button-decrement"
    >
      -
    </button>
    <div
      className="food-item-row__quantity"
    />
    <div
      className="food-item-row__subtotal"
    />
  </div>
</body>
`;
